src/services/auth/auth-google-service.js

```
@@ -54,27 +61,26 @@ export const authGoogleService = {
         // Check if we have a connection
         const hasInternetConnection = await utilsService.hasInternetConnection();
         if (!hasInternetConnection) {
-            notificationService.showAlert(STRINGS[language].status_codes.ec5_118);
+            await notificationService.showAlert(STRINGS[language].status_codes.ec5_118);
         } else {
 
             await notificationService.showProgressDialog(labels.sign_in + '...');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)